New patch. Fix underquoted definition of AM_PATH_LIBXOSD in libxosd.m4 (Closes: #286156).
diff -puriN xosd-2.2.14/libxosd.m4 xosd-2.2.14.new/libxosd.m4
--- xosd-2.2.14/libxosd.m4	2004-08-19 22:39:40.000000000 +0000
+++ xosd-2.2.14.new/libxosd.m4	2007-10-25 15:52:22.000000000 +0000
@@ -6,7 +6,7 @@
 dnl AM_PATH_LIBXOSD([ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND]])
 dnl Test to see if libxosd is installed, and define LIBXOSD_CFLAGS, LIBXOSDLIBS
 dnl
-AC_DEFUN(AM_PATH_LIBXOSD,
+AC_DEFUN([AM_PATH_LIBXOSD],
 [dnl
 dnl Get the cflags and libraries from the xosd-config script
 dnl
